conf/routes

```diff
@@ -22,6 +22,10 @@ GET    /questions/search/:keyString                     controllers.QuestionCont
 GET    /numSols                                         controllers.QuestionController.numberOfGoodQualitySolutions()
 GET    /contributors                                    controllers.QuestionController.distinctContributors()
 
+# For contributors
```

Comment:
@wayneyu Could we please restrict the access to these pages to Contributors and SuperUsers?